go/pkg/sdk/sdk_test.go

```diff
@@ -1,37 +1,61 @@
 package sdk
 
 import (
-	// "encoding/json"
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
 	"fmt"
+	"log"
+	"math"
 	"math/big"
 	"os"
 	"strconv"
+	"strings"
 	"testing"
 
 	"github.com/NovaSubDAO/nova-sdk/go/pkg/constants"
+	ethereumContracts "github.com/NovaSubDAO/nova-sdk/go/pkg/sdk/ethereum/abis"
+	optimismContracts "github.com/NovaSubDAO/nova-sdk/go/pkg/sdk/optimism/abis"
+	"github.com/NovaSubDAO/nova-sdk/go/pkg/utils"
+	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/stretchr/testify/assert"
 )
 
 type testCase struct {
-	rpcEndpoint string
-	chainId     int64
-	stable      constants.Stablecoin
-	mockAddress common.Address
+	rpcEndpoint   string
+	chainId       int64
+	vaultAddress  string
+	vaultDecimals int64
+	stable        constants.Stablecoin
+	sDai          common.Address
+	mockAddress   common.Address
```

Comment:
name it `testAddress`. mock is usually used for constant variables (like 0x123), not for something we pass using an env variable